.github/workflows/ex_doc_version_check.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ex_doc_version_check
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+
+
+jobs:
+  test:
+    name: Check ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get current version
+        run: |
+            CURR=`curl -s https://api.github.com/repos/elixir-lang/ex_doc/tags | \
+                  grep "\"name\":" | \
+                  sort --reverse | \
+                  head -n 1 | \
+                  awk '{print substr($2, 2, length($2)-3)}'`
```

Comment:
This is probably safe, but if `ex_doc` ever releases a patch to an older version, there's no guarantee the latest tag is the "highest" in terms of versioning...

Reply:
Well, yes, but as long as vMajor.minor.patch naming convention is used, it should be safe. We can utilize grep to only match versions which follow the convention

Reply:
(mind you this API call's response is also paginated, getting only the first 30 results, as-is)

Reply:
Hmm, do we need more than the first page? But yeah, I completely missed pagination in this PR

Reply:
> more than the first page?

Theoretically, yes; in practice I don't know. I just wanted to point out a potential issue.

Reply:
I can’t come up with an example where we would need them. I think we’re safe as long as the first page contains newest tags and if ex_doc doesn’t publish 2 releases in a day (in which case we’d miss one)

Reply:
The example would be `ex_doc` to publish 30+ support versions for older tags (when paginating GitHub doesn't care about semver or whatever is "highest", just "more recent"). In that case (unlikely, but still...) even if your comparison were using semver it would fail to recognise a new version. And since I can't be sure there won't be 30+ support versions... now that I think about it, is there no way to get "latest"? That'd solve the pagination issue, too.

Reply:
Then we can increase per-page entries to considerably large number (e.g. 100) and sort tags by their name.

Reply:
okay, per-page parameter can me from 1-100, but I think it should do the job even if is not perfect...  Worst it can do is a false-positive issue that can be closed

Reply:
I'm Ok with that.